packages/cli/src/commands/element.ts

```diff
@@ -736,7 +738,28 @@ type PrintElementArgs = {
   selectors: string[]
   source: 'nacl' | 'state'
   onlyValue: boolean
+  format?: 'nacl' | 'inspect' | 'serialize'
```

Comment:
I think we always have a format since we gave the option a default value

```suggestion
  format: 'nacl' | 'inspect' | 'serialize'
```